ui/components/src/LineChart/LineChart.tsx

```diff
@@ -86,6 +87,7 @@ export function LineChart({
   grid,
   legend,
   tooltipConfig = { wrapLabels: true },
+  noDataVariant = 'message',
```

Comment:
I went with this prop name because it's managing different variations of display when no data is available, but I'm option to other suggestions.

Reply:
what about `noDataDisplay`?

Reply:
ok with `noDataVariant` too